phpBB/adm/style/acp_ban.html

```diff
@@ -52,7 +52,7 @@
 	<legend>{L_TITLE}</legend>
 <dl>
 	<dt><label for="ban">{L_BAN_CELL}{L_COLON}</label></dt>
-	<dd><textarea name="ban" cols="40" rows="3" id="ban"></textarea></dd>
+	<dd><!-- EVENT acp_ban_cell_textarea_prepend --><textarea name="ban" cols="40" rows="3" id="ban"></textarea><!-- EVENT acp_ban_cell_textarea_append --></dd>
```

Comment:
I'd prefer an event with a more semantic name (not so tied to the html structure)